src/worker/coverage.ts

```diff
@@ -82,7 +83,7 @@ export class VitestCoverage {
   async waitForReport() {
     if (!this.enabled)
       return null
-    const coverage = this.ctx.config.coverage
+    const coverage = this.config
```

Comment:
`this.ctx.config.coverage` should already reference `this.config` (see the override at line 19)

Reply:
Agreed, will revert this.